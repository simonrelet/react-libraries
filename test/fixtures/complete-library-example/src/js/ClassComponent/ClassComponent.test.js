import { render } from '@testing-library/react'
import React from 'react'
import { ClassComponent } from './ClassComponent'

describe('<ClassComponent />', () => {
  it('should render the component', () => {
    const { container } = render(<ClassComponent />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
