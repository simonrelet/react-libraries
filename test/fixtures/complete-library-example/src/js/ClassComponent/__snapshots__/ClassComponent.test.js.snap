// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClassComponent /> should render the component 1`] = `
<p>
  ClassComponent
</p>
`;
